bin/git-forgit

```diff
@@ -247,8 +247,11 @@ _forgit_cherry_pick() {
 
 _forgit_cherry_pick_from_branch() {
     _forgit_inside_work_tree || return 1
-    [[ $# -ne 0 ]] && { git checkout -b "$@"; return $?; }
-    local cmd preview opts branch exitval
+    local cmd preview opts branch exitval input_branch args
+    args=("$@")
+    if [[ $# -ne 0 ]] && [[ $(git rev-parse --verify "${args[0]}" 2>/dev/null) ]]; then
```

Comment:
I don't think that this function needs to check if the branch exists. In fact I find it even confusing when forgit silently ignores my argument if it does not exist. So I would suggest to remove the second half of the if-condition here and leave the error handling to the actual `git` git call.